src/components/dialogs/NameRequestErrorDialog.vue

```diff
@@ -12,11 +12,15 @@
 
         <p v-else-if="type === NameRequestStates.CONSUMED">The specified name request has already been consumed.</p>
 
-        <template v-else-if="type === NameRequestStates.NOT_FOUND || type === NameRequestStates.INVALID ||
-                  type === NameRequestStates.INCORRECT_EMAIL || type === NameRequestStates.INCORRECT_PHONE">
-          <p>We could not find a match for the information you have entered.
-          Please verify the NR Number and the phone number or email address and try again.</p>
-        </template>
+        <p v-else-if="type === NameRequestStates.NOT_FOUND ||
+                      type === NameRequestStates.INCORRECT_EMAIL ||
+                      type === NameRequestStates.INCORRECT_PHONE"
+        >
+          We could not find a match for the information you have entered.
+          Please verify the NR Number and the phone number or email address and try again.
+        </p>
+
+        <p v-else-if="type === NameRequestStates.INVALID">The specified name request is not valid.</p>
```

Comment:
I added this case because, during testing, I encountered some invalid NRs, and the previous message was "could not find a match". This makes more sense (and probably won't be encountered in Prod anyway).